### PR TITLE
Add QLMobi

### PR DESCRIPTION
**Pull request by pkuczynski**

### CODE REVIEW

**Comment by sindresorhus** [comment]
This plugin is not really useful for developers specifically.

**Comment by Toxblh** [comment]
> This plugin is not really useful for developers specifically.

Is it problem? The repo I recommend freq as best pack for improve your QL in mac, and this not only developers. And this widget will be helpful for me as developer as well. For quick look inside any e-book files on my mac.

**Comment by pkuczynski** [comment]
Yeah, had the same thoughts as @Toxblh - this list does not say "QL Plugings for Devs"... Just a general list of awesome ql plugins, right?

**Comment by sindresorhus** [comment]
> this list does not say "QL Plugings for Devs"...

It does actually say that in the repo description and the top of the readme.

**Comment by sindresorhus** [comment]
But I buy the argument that it can be useful for devs for programming books, etc.

File: readme.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 #### Install all
 
 ```
-brew cask install qlcolorcode qlstephen qlmarkdown quicklook-json qlimagesize suspicious-package quicklookase qlvideo
+brew cask install qlcolorcode qlstephen qlmarkdown quicklook-json qlimagesize webpquicklook suspicious-package quicklookase qlvideo qlmobi
```
Comment (sindresorhus) [inline readme.md]: Remove it from here and don't do unrelated changes.

File: readme.md [hunk 2]
```diff
@@ -110,6 +110,15 @@ Run `brew cask install qlvideo` or [download manually](https://github.com/Margin
 [![](screenshots/QLVideo.png)](https://github.com/Marginal/QLVideo)
 
 
+### [QLMobi](https://github.com/bfabiszewski/QLMobi)
```
Comment (sindresorhus) [inline readme.md]: Move it to the `More` section.

File: readme.md [hunk 3]
```diff
@@ -110,6 +110,15 @@ Run `brew cask install qlvideo` or [download manually](https://github.com/Margin
...
+
+> Preview various ebook formats used on Kindle readers (prc, mobi, azw, azw3, azw4 and some pdb files)
+
+Run `brew cask install qlmobi` or [download manually](https://github.com/bfabiszewski/QLMobi/releases/latest)
+
+[![](screenshots/QLMobi.jpg)](https://github.com/bfabiszewski/QLMobi)
```
Comment (sindresorhus) [inline readme.md]: Needs a better screenshot. Pick a page in the book without an image, and take the a window screenshot (with shadows) as PNG and in 2x retina.

**Comment by sindresorhus** [comment]
Bump
